### ISSUE TITLE
API change broke app

### ISSUE THREAD

**Issue by camjac251** [HugoJF/twitch-clip-downloader#12]
I tried to use this software again today after having used it back in June/July but I think a twitch API change has broken the app from working.

I get this error when I try to export from a channel

```
⠹ Paginating API, please wait...(node:1355) UnhandledPromiseRejectionWarning: Error: Request failed with status code 401
    at createError (/path/twitch-clip-downloader/node_modules/axios/lib/core/createError.js:16:15)
    at settle (/path/twitch-clip-downloader/node_modules/axios/lib/core/settle.js:17:12)
    at IncomingMessage.handleStreamEnd (/path/twitch-clip-downloader/node_modules/axios/lib/adapters/http.js:236:11)
    at IncomingMessage.emit (events.js:326:22)
    at endReadableNT (_stream_readable.js:1252:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:1355) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:1355) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
⠏ Paginating API, please wait...
```

**Comment by HugoJF** [comment]
Interesting, I'll take a look at the docs tonight to try to figure out what happened.

**Comment by camjac251** [comment]
Do you have discord by the way? I could help troubleshoot if need be and give logs

**Comment by camjac251** [comment]
Have you seen this https://twitter.com/CommanderRoot/status/1269883095446159362

I think it is able to get many more results. I saw a stream of someone deleting 116,000+ clips in the matter of two years with the tool but I could only grab 40,000-50,000 on some channels using this repo. It could be helpful to find the most clips

**Comment by HugoJF** [comment]
> Do you have discord by the way? I could help troubleshoot if need be and give logs

Yes, [https://denerdtv.com/discord](https://denerdtv.com/discord).

> Have you seen this https://twitter.com/CommanderRoot/status/1269883095446159362
> 
> I think it is able to get many more results. I saw a stream of someone deleting 116,000+ clips in the matter of two years with the tool but I could only grab 40,000-50,000 on some channels using this repo. It could be helpful to find the most clips

Well, this tool uses the internal API (that's why it runs inside the browser with TamperMonkey) which is different from the "public" API Twitch created.

I've tested with some channels and was able to get everything with this repo, so I'm not really sure why there's a difference.

The previous version of this repo used an endpoint that only got top clips but that was fixed with the new API version. This tool get's everything that Twitch reports via the API, if somehow something is missing I can't really do much.

I've considered implementing this using their internal API, but decided against it since it's technically against terms of service and I figured most people would get scared of providing their authentication token to the tool.

Well,` youtube-dl` got DCMA'd and I don't have a local installation of the `node-youtube-dl`, that's annoying...
